library/core/src/iter/traits/iterator.rs

```diff
@@ -1332,7 +1332,7 @@ pub trait Iterator {
     /// assert_eq!(merged, "alphabetagamma");
     /// ```
     ///
-    /// Flattening once only removes one level of nesting:
+    /// Flattening only removes one level of nesting at a time:
```

Comment:
```suggestion
    /// Flattening only removes one level of nesting once:
```

"at a time" is not clear that it happened once, feels like a play a words, people may not notice it if not careful.